### ISSUE TITLE
The new status icon does not call attention enough when blinking

### ISSUE THREAD

**Issue by clopez** [aperezdc/revolt#28]
The new status icon (added on 77c972820c1b977e3c4caf0ba9a0db290433d404) does not call attention enough when it blinks (at least on my XFCE desktop).

Currently the surface that changes color is only a dot in the top-left corner.

I think that we should change that and make the surface that changes color much bigger (as bigger as possible). It is also a good idea to use very different colours between states (like green and red)

The idea is to be sure that the user always pays attention to the blinking icon.

**Comment by aperezdc** [comment]
While I understand the rationaly, precisely the current icon was made more discreet because the older one was too loud. Maybe a good option for this is providing a setting to let the user choose between:

1. No status icon.
2. Discreet status icon.
3. “HEY GIMME YOUR ATTENTION” status icon.

**Comment by clopez** [comment]
I have to say that I got used to the new icon, and I have ended appreciating  that it is not as loud as the previous one. As this allows one to not be distracted as much as with the other. Also the whole point of matrix is that is an async chat platform. There is no need to receive notifications as in real-time as with IRC.

I'm closing this as wontfix.
If anyone else wants to implement this,  feel free to do it and send a PR. I won't do it.
